### ISSUE TITLE
Way to add a drop shadow?

### ISSUE THREAD

**Issue by arngo** [micheleg/dash-to-dock#661]
Is there a way to add a drop shadow with a CSS rule? I have tried:

 `#dashtodockContainer #dash { box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.7); }` 

with no luck.

**Comment by micheleg** [comment]
Hi,
sorry for the late reply. The property is supported by the shell. Unfortunately with the sliding machinery we are clipping what is shown: 
https://github.com/micheleg/dash-to-dock/blob/fd81051ee0ae8320ef52eeeca3091d6f8f4860be/docking.js#L138-L139

and
https://github.com/micheleg/dash-to-dock/blob/fd81051ee0ae8320ef52eeeca3091d6f8f4860be/dash.js#L74

I deleted those lines and the shadow is shown. Do we need this clipping? I need to investigate. If you don't mind deleting those lines and see if any unwanted behavior occurs, that  would be helpful.

**Comment by arngo** [comment]
Sorry for the very late reply as well, I was on vacation. The edge of the shadow is visible when the dock is hidden, but it is very subtle if the shadow is not too dark. Can the shadow be faded out or simply disappear when the dock is hidden?

**Comment by micheleg** [comment]
Yes, I was expecting this sort of problems. I think in principle it could be fixed to allow for such styling but it could require to rework the way the dock is layout, so not something I can improve straight away.
